api/import.go

```diff
@@ -46,16 +46,18 @@ type ImportHandler struct {
 //
 // AddRoutes adds routes.
 func (h ImportHandler) AddRoutes(e *gin.Engine) {
-	e.GET(SummariesRoot, h.ListSummaries)
-	e.GET(SummariesRoot+"/", h.ListSummaries)
-	e.GET(SummaryRoot, h.GetSummary)
-	e.DELETE(SummaryRoot, h.DeleteSummary)
-	e.GET(ImportsRoot, h.ListImports)
-	e.GET(ImportsRoot+"/", h.ListImports)
-	e.GET(ImportRoot, h.GetImport)
-	e.DELETE(ImportRoot, h.DeleteImport)
-	e.GET(DownloadRoot, h.DownloadCSV)
-	e.POST(UploadRoot, h.UploadCSV)
+	routeGroup := e.Group("/")
+	routeGroup.Use(Required("imports"), Transaction)
```

Comment:
Recommend Require("applications") instread of imports.  The applications scopes already exist and mapped to users.
Also, the new `Transaction` will not be on the mta-6.1 branch.

Reply:
The "imports" scope exists as well and is already mapped, but we could consolidate it with the "applications" scope. Good catch on the branch compatibility issue.

Reply:
I was thinking about the application since this endpoint main purpose is to create applications (but found out it exists, so used it), will update to "applications". Good catch on Transaction!

(potential deletion of "imports" scope might be part of different PR)

Reply:
I didn't check if _imports_ existed and wanted to avoid creating it.
Let's keep it as _imports_.  Sorry for the confusion.